Library/Strings.swift

```diff
@@ -2269,18 +2269,18 @@ Haz clic para volver a intentarlo."
     )
   }
   /**
-   "Continue with %{quantity_count} add-ons"
+   "Continue with %{quantity_count} add-on"
```

Comment:
These are weird. Did something happen to muck with plurals?

Reply:
I've seen similar changes before that weirded me out. Basically, we have multiple versions of this string depending on the count. The change here is in the comment that's just for our sake; not in the actual string presented to users. I have no idea what makes the comment choose the singular vs plural version, though. I assume the change to the default value is similar (not actually going to have an effect since we're not relying on the default), but I'm not sure.

Reply:
Ooh, that makes much more sense. Cool!